### ISSUE TITLE
How to train model in Tensorflow for multi class Object Detection using large MS COCO Dataset?

### ISSUE THREAD

**Issue by spurani** [sglvladi/TensorFlowObjectDetectionTutorial#56]
I am trying to train my model on custom dataset using this https://github.com/tensorflow/models/tree/master/research/object_detection as a reference which you covered in your tutorial. Basically I am having Acer Nitro 50 Desktop with system configuration Processor: Intel® Core™ i5-8400 CPU @ 2.80GHz × 6, Graphics: GeForce GTX 1050/PCIe/SSE2 (2 GB), Memory(RAM): 8GB DDR4 Memory

I am working with tensorflow 1.12.0 gpu | bazel 0.15.0 | python 3.5 | GCC 4.8 | cudnn 7 | Cuda 9.0 to train a faster_rcnn_inception_v2_coco model on my custom ms coco dataset with 3.7 GB(20952 images) of training data and 43.7 MB(2328 images) of validation data for object detection for 200k epochs(num_steps it will be training at 600 x 1024 default resolution which is set in faster_rcnn_inception_v2_coco.config ). I am training and validating model on  8 classes. So when I am training the model the accuracy(map) is not increasing and loss is not decreasing after a while. After the completion of the training successfully I ran the model on various images and noticed couple of things.

**Dataset**
I have completely balanced dataset I have same number of images and labels for each category. I have in total 8 categories classes (cabinet,counter,floor-other,furniture-other,wall-concrete,wall-tile,dog,cat). The images and their respective labels has been downloaded from MS COCO dataset

**Cause of Issue**
But the label's files also have couple other classes as well along with the classes which I specifically want to detect. 
[000000000089.txt](https://github.com/sglvladi/TensorFlowObjectDetectionTutorial/files/5286107/000000000089.txt)
![000000000089](https://user-images.githubusercontent.com/28580888/94328718-bbcfa380-ff82-11ea-997f-8838a8a42c75.jpg)
As you can see in above label txt file category classes concerned to my needs are(cabinet,counter,floor-other,furniture-other,wall-concrete,wall-tile) and I don't care about rest of the classes. So in such manner there are 8 categories classes with labels files for each category having class several other class categories which I don't need at all. what could be done at this point of scenario?

I tried to train my model with different pretrained models such as ssd. but did'nt work. I think those extra labels are causing an issue to not decrease the loss. Can you please guide me through this?
![image](https://user-images.githubusercontent.com/28580888/94329692-4caa7d00-ff8b-11ea-8852-0e2dcaaa8c29.png)
Here is total loss image at specific step.

It would be really helpful if anyone can guide me as I will be able to find out something more and interesting with new learnings. I am trying to figure out this issue from a while. I am not getting any errors or warnings. Please feel free to reach out should you have any questions
